Courses/Views/Screens/MatchedView.swift

```diff
@@ -0,0 +1,74 @@
+//
+//  MatchedView.swift
+//  Courses
+//
+//  Created by Dmytro Churakov on 13.10.2022.
+//
+
+import SwiftUI
+
+struct MatchedView: View {
+    @Namespace var namespace
+    @State var show = false
+    
+    
+    var body: some View {
+        ZStack {
+            if !show {
+                VStack(alignment: .leading, spacing: 12) {
```

Comment:
Here is better to create a separate view for Course

Reply:
@yura-hb , It was just created and I haven't finalized it yet. But I also assume that I did not fully understand what the error is.

Reply:
I meant to just create a separate view instead of writing this:

```
 VStack(alignment: .leading, spacing: 12) {
                     Spacer()
                     Text("Build a SwiftUI app for iOS 15")
                         .font(.largeTitle.weight(.bold))
                         .lineLimit(1)
                         .matchedGeometryEffect(id: "title", in: namespace)
                         .frame(maxWidth: .infinity, alignment: .leading)
                     .frame(maxWidth: .infinity, alignment: .leading)
                     Text("20 sections - 3 hours".uppercased())
                         .font(.footnote.weight(.semibold))
                         .matchedGeometryEffect(id: "subtitle", in: namespace)
                     Text("Design and code a SwiftUI 3 app with custom layouts, animations and gestures using Xcode 13, SF Symbols 3, Canvas, Concurrency, Searchable and a whole lot more")
                         .font(.footnote)
                         .lineLimit(1)
                         .matchedGeometryEffect(id: "text", in: namespace)
                 }
```

Reply:
Ok, got it, thx